src/rust/engine/rule_graph/README.md

```diff
@@ -83,6 +83,7 @@ async def fibonacci(n: int) -> Fibonacci:
     x, y = await Get(Fib, int(n - 2)), await Get(Fib, int(n - 1))
     return Fibonacci(x.val + y.val)
 ```
+
 ```
```

Comment:
Add `python` to this, and Readme.io will make it look much prettier. Ditto on the above.